esmvaltool/config-references.yml

```
@@ -30,6 +30,12 @@ authors:
     email: diego.cammarano@dlr.de
     github: diegokam
     orcid:
+  castellani_giulia
+    name: Castellani, Giulia
+    institute: University of Bremen
+    email: giulia.castellani@uni-bremen.de
+    github: giuliacast
+    orcid: 
```

```suggestion
    orcid: https://orcid.org/0000-0001-6151-015X
```

You specified it above, so it would be nice to also have it here 😊